pyproject.toml

```diff
@@ -170,30 +177,6 @@ force-sort-within-sections = true
   "E501", # Line too long being fixed in #1746 To be removed after merging
 ]
 
-[tool.black]
-line-length = 88
-include = '\.pyi?$'
-exclude = '''
-(
-  /(
-      \.eggs         # exclude a few common directories in the
```

Comment:
where did this go? Explanation appreciated.

Reply:
it is in the Ruff section